qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -461,10 +461,12 @@ def test_du_works_for_empty_dirs(self):
     def test_du_works_for_hardlinks(self):
         regfilename = 'some_regfile'
         regfile_abspath = path.join(self.mount_a.mountpoint, regfilename)
-        sudo_write_file(self.mount_a.client_remote, regfile_abspath, 'somedata')
+        sudo_write_file(self.mount_a.client_remote, regfile_abspath,
+                        'somedata')
```

Comment:
The issue likely is that this test is creating a file as root. You can't do a hardlink if you're not the same user. So the real question is: why is it creating this file with sudo?

Reply:
Nice point. Replacing `sudo_write_file` throughout this TestDU...

Reply:
I am going ahead with this, but I slightly suspect that I might need superuser privileges on the "test" machines to write a file in to `/tmp` directory and CephFS mounts.

Reply:
Blah, I forgot `run_shell` prefixes everything with `sudo`.

Maybe it's time to add a uid/gid setting to cephfs-shell. Then it can mount CephFS as the correct user (uid==0).